### ISSUE TITLE
[wxWidgets] build failure on github actions (Windows-lastest)

### ISSUE THREAD

**Issue by Stovent** [microsoft/vcpkg#26849]
**Host Environment**

- OS: Windows-lastest as in github actions
- Compiler: Mingw-w64 11.2.0
all information about the OS and pre-installed software should be listed here : https://github.com/actions/runner-images/blob/main/images/win/Windows2022-Readme.md

**To Reproduce**

The command issued in my workflow is `vcpkg install wxwidgets:x64-mingw-static`.
You can find the full workflow and the command outputs here : https://github.com/Stovent/CeDImu/actions/runs/3063817118/jobs/4946307979

**Failure logs**

```
[1/815] C:\ProgramData\chocolatey\bin\x86_64-w64-mingw32-gcc.exe -DUNICODE -DWXBUILDING -DWX_PRECOMP -D_UNICODE -D__WXMSW__ -DwxUSE_BASE=1 -DwxUSE_GUI=0 -IC:/vcpkg/installed/x64-mingw-static/debug/../include -IC:/vcpkg/installed/x64-mingw-static/include -IC:/vcpkg/buildtrees/wxwidgets/x64-mingw-static-dbg/lib/mswud -IC:/vcpkg/buildtrees/wxwidgets/src/v3.2.0-8a0adfb316.clean/include -g -Wall -Wno-ctor-dtor-privacy -Woverloaded-virtual -Wundef -Wunused-parameter -pthread -MD -MT libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/common/extended.c.obj -MF libs\base\CMakeFiles\wxbase.dir\__\__\__\__\src\common\extended.c.obj.d -o libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/common/extended.c.obj -c C:/vcpkg/buildtrees/wxwidgets/src/v3.2.0-8a0adfb316.clean/src/common/extended.c
cc1.exe: warning: command-line option '-Wno-ctor-dtor-privacy' is valid for C++/ObjC++ but not for C
cc1.exe: warning: command-line option '-Woverloaded-virtual' is valid for C++/ObjC++ but not for C
[2/815] C:\ProgramData\chocolatey\bin\x86_64-w64-mingw32-g++.exe -DUNICODE -DWXBUILDING -DWX_PRECOMP -D_UNICODE -D__WXMSW__ -DwxUSE_BASE=0 -DwxUSE_GUI=1 -IC:/vcpkg/buildtrees/wxwidgets/x64-mingw-static-dbg/lib/mswud -IC:/vcpkg/buildtrees/wxwidgets/src/v3.2.0-8a0adfb316.clean/include -g -Wall -Wno-ctor-dtor-privacy -Woverloaded-virtual -Wundef -Wunused-parameter -pthread -MD -MT libs/adv/CMakeFiles/wxadv.dir/__/__/__/__/src/common/dummy.cpp.obj -MF libs\adv\CMakeFiles\wxadv.dir\__\__\__\__\src\common\dummy.cpp.obj.d -o libs/adv/CMakeFiles/wxadv.dir/__/__/__/__/src/common/dummy.cpp.obj -c C:/vcpkg/buildtrees/wxwidgets/src/v3.2.0-8a0adfb316.clean/src/common/dummy.cpp
[3/815] C:\ProgramData\chocolatey\bin\x86_64-w64-mingw32-g++.exe -DUNICODE -DWXBUILDING -DWX_PRECOMP -D_UNICODE -D__WXMSW__ -DwxUSE_BASE=1 -DwxUSE_GUI=0 -IC:/vcpkg/installed/x64-mingw-static/debug/../include -IC:/vcpkg/installed/x64-mingw-static/include -IC:/vcpkg/buildtrees/wxwidgets/x64-mingw-static-dbg/lib/mswud -IC:/vcpkg/buildtrees/wxwidgets/src/v3.2.0-8a0adfb316.clean/include -g -Wall -Wno-ctor-dtor-privacy -Woverloaded-virtual -Wundef -Wunused-parameter -pthread -Winvalid-pch -x c++-header -include C:/vcpkg/buildtrees/wxwidgets/x64-mingw-static-dbg/libs/base/CMakeFiles/wxbase.dir/cmake_pch.hxx -MD -MT libs/base/CMakeFiles/wxbase.dir/cmake_pch.hxx.gch -MF libs\base\CMakeFiles\wxbase.dir\cmake_pch.hxx.gch.d -o libs/base/CMakeFiles/wxbase.dir/cmake_pch.hxx.gch -c C:/vcpkg/buildtrees/wxwidgets/x64-mingw-static-dbg/libs/base/CMakeFiles/wxbase.dir/cmake_pch.hxx.cxx
[4/815] C:\ProgramData\chocolatey\bin\x86_64-w64-mingw32-g++.exe -DUNICODE -DWXBUILDING -DWX_PRECOMP -D_UNICODE -D__WXMSW__ -DwxUSE_BASE=1 -DwxUSE_GUI=0 -IC:/vcpkg/installed/x64-mingw-static/debug/../include -IC:/vcpkg/installed/x64-mingw-static/include -IC:/vcpkg/buildtrees/wxwidgets/x64-mingw-static-dbg/lib/mswud -IC:/vcpkg/buildtrees/wxwidgets/src/v3.2.0-8a0adfb316.clean/include -g -Wall -Wno-ctor-dtor-privacy -Woverloaded-virtual -Wundef -Wunused-parameter -pthread -Winvalid-pch -include C:/vcpkg/buildtrees/wxwidgets/x64-mingw-static-dbg/libs/base/CMakeFiles/wxbase.dir/cmake_pch.hxx -MD -MT libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/common/appbase.cpp.obj -MF libs\base\CMakeFiles\wxbase.dir\__\__\__\__\src\common\appbase.cpp.obj.d -o libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/common/appbase.cpp.obj -c C:/vcpkg/buildtrees/wxwidgets/src/v3.2.0-8a0adfb316.clean/src/common/appbase.cpp
FAILED: libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/common/appbase.cpp.obj 
C:\ProgramData\chocolatey\bin\x86_64-w64-mingw32-g++.exe -DUNICODE -DWXBUILDING -DWX_PRECOMP -D_UNICODE -D__WXMSW__ -DwxUSE_BASE=1 -DwxUSE_GUI=0 -IC:/vcpkg/installed/x64-mingw-static/debug/../include -IC:/vcpkg/installed/x64-mingw-static/include -IC:/vcpkg/buildtrees/wxwidgets/x64-mingw-static-dbg/lib/mswud -IC:/vcpkg/buildtrees/wxwidgets/src/v3.2.0-8a0adfb316.clean/include -g -Wall -Wno-ctor-dtor-privacy -Woverloaded-virtual -Wundef -Wunused-parameter -pthread -Winvalid-pch -include C:/vcpkg/buildtrees/wxwidgets/x64-mingw-static-dbg/libs/base/CMakeFiles/wxbase.dir/cmake_pch.hxx -MD -MT libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/common/appbase.cpp.obj -MF libs\base\CMakeFiles\wxbase.dir\__\__\__\__\src\common\appbase.cpp.obj.d -o libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/common/appbase.cpp.obj -c C:/vcpkg/buildtrees/wxwidgets/src/v3.2.0-8a0adfb316.clean/src/common/appbase.cpp

[5/815] C:\ProgramData\chocolatey\bin\x86_64-w64-mingw32-g++.exe -DUNICODE -DWXBUILDING -DWX_PRECOMP -D_UNICODE -D__WXMSW__ -DwxUSE_BASE=0 -DwxUSE_GUI=1 -IC:/vcpkg/buildtrees/wxwidgets/x64-mingw-static-dbg/lib/mswud -IC:/vcpkg/buildtrees/wxwidgets/src/v3.2.0-8a0adfb316.clean/include -g -Wall -Wno-ctor-dtor-privacy -Woverloaded-virtual -Wundef -Wunused-parameter -pthread -Winvalid-pch -x c++-header -include C:/vcpkg/buildtrees/wxwidgets/x64-mingw-static-dbg/libs/aui/CMakeFiles/wxaui.dir/cmake_pch.hxx -MD -MT libs/aui/CMakeFiles/wxaui.dir/cmake_pch.hxx.gch -MF libs\aui\CMakeFiles\wxaui.dir\cmake_pch.hxx.gch.d -o libs/aui/CMakeFiles/wxaui.dir/cmake_pch.hxx.gch -c C:/vcpkg/buildtrees/wxwidgets/x64-mingw-static-dbg/libs/aui/CMakeFiles/wxaui.dir/cmake_pch.hxx.cxx
[6/815] C:\ProgramData\chocolatey\bin\x86_64-w64-mingw32-g++.exe -DUNICODE -DWXBUILDING -DWX_PRECOMP -D_UNICODE -D__WXMSW__ -DwxUSE_BASE=0 -DwxUSE_GUI=1 -IC:/vcpkg/buildtrees/wxwidgets/x64-mingw-static-dbg/lib/mswud -IC:/vcpkg/buildtrees/wxwidgets/src/v3.2.0-8a0adfb316.clean/include -g -Wall -Wno-ctor-dtor-privacy -Woverloaded-virtual -Wundef -Wunused-parameter -pthread -Winvalid-pch -x c++-header -include C:/vcpkg/buildtrees/wxwidgets/x64-mingw-static-dbg/libs/html/CMakeFiles/wxhtml.dir/cmake_pch.hxx -MD -MT libs/html/CMakeFiles/wxhtml.dir/cmake_pch.hxx.gch -MF libs\html\CMakeFiles\wxhtml.dir\cmake_pch.hxx.gch.d -o libs/html/CMakeFiles/wxhtml.dir/cmake_pch.hxx.gch -c C:/vcpkg/buildtrees/wxwidgets/x64-mingw-static-dbg/libs/html/CMakeFiles/wxhtml.dir/cmake_pch.hxx.cxx
ninja: build stopped: subcommand failed.
```

[install-x64-mingw-static-dbg-out.log](https://github.com/microsoft/vcpkg/files/9593043/install-x64-mingw-static-dbg-out.log)

If you need additional information let me know, I can modify and run other workflows to help diagnose the problem.

**Comment by JonLiu1993** [comment]
This issue hasn’t been updated in 3 month, if it is still an issue, please reopen this issue.

**Comment by Stovent** [comment]
The issue is still here, I've run a new workflow to confirm : https://github.com/Stovent/CeDImu/actions/runs/4580589282
Unfortunately the new log file doesn't contain more information and I can't reproduce it locally so I don't know how this could be troubleshooted.

Also I cannot reopen the issue myself

**Comment by JonLiu1993** [comment]
This issue hasn’t been updated in 3 month, if it is still an issue, please reopen this issue.

**Comment by Stovent** [comment]
For reference this is now fixed when installing wxWidgets 3.2.3 (don't know what fixed it though) : https://github.com/Stovent/CeDImu/actions/runs/7001405993/job/19043459349
